torch_tvm/fuse_linear.cpp

```diff
@@ -0,0 +1,50 @@
+#include "fuse_linear.h"
+#include <torch/csrc/jit/passes/subgraph_rewrite.h>
+
+// This pass fuse the addmm or matmul + add generated by JIT back to linear
+// to allow direct support with tvm integration with Relay IR
+// This pass can be deleted once the JIT can emit the aten::linear in the future
+void FuseLinear(std::shared_ptr<Graph>& graph) {
+    std::string addmm_pattern = R"IR(
+    graph(%input, %weight, %bias, %4):
+        %weight_t = aten::t(%weight)
+        %res = aten::addmm(%bias, %input, %weight_t, %4, %4)
+        return (%res))IR";
+    std::string matmul_add_pattern = R"IR(
+    graph(%input, %weight, %bias, %4):
+        %weight_t = aten::t(%weight)
+        %output = aten::matmul(%input, %weight_t)
+        %res = aten::add_(%output, %bias, %4)
```

Comment:
should we also match on non-inplace add?

Reply:
We can but I don't think we need, people using `nn.Linear` will always get this inplace `add` through tracing/scripting, outplace `add` will be matched if people manually write `matmul + outplace add`.